careers/_template-students-and-graduates.html

```diff
@@ -0,0 +1,9 @@
+<h2 class="h4">Students & Recent Graduates</h2>
+<p class="short-desc">
+    Find opportunities in public service for a new generation of leaders.
+</p>
+<p class="short-desc">
+    <a href="/careers/students-and-graduates/" class="jump-link jump-link__right">
+        Learn about students & graduates opportunities
```

Comment:
:warning: Drive by content opinion incoming! :car: :open_mouth: 

Seems like it should be "Learn about student & graduate opportunities", no? The "recent" seems unnecessary in this link, given its presence in the heading above.